WootricSDK/WootricSDK/WTRColor.m

```diff
@@ -31,7 +31,7 @@ + (UIColor *)viewBackgroundColor {
 }
 
 + (UIColor *)dismissXColor {
-  return [self colorWithHexString:@"#C8C8C8"];
+  return [self colorWithHexString:@"#253746"];
```

Comment:
Looks like this change makes it harder to see the button https://github.com/Wootric/WootricSDK-iOS/issues/90